workflow-dev/tpl/deis-registry-secret.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: registry-secret
+  namespace: deis
+  labels:
+    heritage: deis
+  annotations:
+    deis.io/registry-location: "{{ or (env "REGISTRY_LOCATION") (.registry_location)}}"
```

Comment:
Nitpick: could you put a space before the closing double-braces `}}`? I think go templating reads more easily that way but we're not being consistent here, or on some of the following lines.